### ISSUE TITLE
Some Gitlab badges incorrectly use /v in route

### ISSUE THREAD

**Issue by chris48s** [badges/shields#8138]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

Some GitLab badges e.g:

https://img.shields.io/gitlab/v/license/gitlab-org/gitlab
https://img.shields.io/gitlab/v/contributor/gitlab-org/gitlab

use `/v` in the route even though they aren't version badges. We missed this in code review :(

### 🔗 Link to the badge

https://img.shields.io/gitlab/v/license/gitlab-org/gitlab
https://img.shields.io/gitlab/v/contributor/gitlab-org/gitlab

### 💡 Possible Solution

Change the routes to remove use the `/v` prefix and set up redirectors to keep legacy badges working

**Comment by calebcartwright** [comment]
I wonder how realistic it might be to introduce a CI check that could provide _some_ degree of route validation. I suspect it would be fairly minimal in terms of utility and fairly maximal in terms of effort + code maintenance so likely not worthwhile, but tossing it out there anyway

**Comment by chris48s** [comment]
Yeah its interesting. There probably are some rules we could express for example:

- if route contains `/l` category should be license
- if route contains `/v` category should be version
- if route contains `/dd`, `/dm`, `/dy` etc category should be downloads
- etc

The easiest place for us to implement something like that as a CI check would be using danger https://github.com/badges/shields/blob/master/dangerfile.js but we may be getting into stuff where we would really need to parse the file into an AST in order to do these checks rather than just looking at high-level stuff like what files are modified in the PR. At that point you're probably looking at something more like an ESLint rule.
